server/migrations/utils/index.js

```diff
@@ -0,0 +1,245 @@
+const { Sequelize } = require('sequelize');
+const {
+    createTestResultId,
+    createScenarioResultId,
+    createAssertionResultId
+} = require('../../services/PopulatedData/locationOfDataId');
+const { hashTests } = require('../../util/aria');
+
+/**
+ * Recompute TestPlanVersion.hashedTests
+ * @param queryInterface - The Sequelize.QueryInterface object.
+ * @param transaction - The Sequelize.Transaction object.
+ * @param testPlanVersionWhere - Additional WHERE clauses when querying TestPlanVersion if needed
+ * See {@https://sequelize.org/api/v6/class/src/sequelize.js~sequelize#instance-method-transaction}
+ * @returns {Promise<void>}
+ */
+const computeTestPlanVersionHashedTests = async (
+    queryInterface,
+    transaction,
+    testPlanVersionWhere
+) => {
+    const results = await queryInterface.sequelize.query(
+        `SELECT COUNT(*) FROM "TestPlanVersion" ${testPlanVersionWhere}`,
+        { transaction }
+    );
+    const [[{ count: testPlanVersionCount }]] = results;
+
+    const testPlanVersionBatchSize = 10;
+    const iterationsNeeded = Math.ceil(
+        testPlanVersionCount / testPlanVersionBatchSize
+    );
+
+    const knownHashedTests = [];
+    for (let i = 0; i < iterationsNeeded; i += 1) {
+        const multipleOf100 = i % testPlanVersionBatchSize === 0;
+        if (multipleOf100)
+            // eslint-disable-next-line no-console
+            console.info(
+                'Indexing TestPlanVersions',
+                i * testPlanVersionBatchSize,
+                'of',
+                Number(testPlanVersionCount)
+            );
+        const currentOffset = i * testPlanVersionBatchSize;
+
+        const [testPlanVersions] = await queryInterface.sequelize.query(
+            `SELECT id, directory, "gitSha", tests, "updatedAt" FROM "TestPlanVersion" ${testPlanVersionWhere} ORDER BY id LIMIT ? OFFSET ?`,
+            {
+                replacements: [testPlanVersionBatchSize, currentOffset],
+                transaction
+            }
+        );
+
+        await Promise.all(
+            testPlanVersions.map(async testPlanVersion => {
+                const hashedTests = hashTests(testPlanVersion.tests);
+
+                if (!knownHashedTests.includes(hashedTests)) {
+                    knownHashedTests.push(hashedTests);
+                    await queryInterface.sequelize.query(
+                        `UPDATE "TestPlanVersion" SET "hashedTests" = ? WHERE id = ?`,
+                        {
+                            replacements: [hashedTests, testPlanVersion.id],
+                            transaction
+                        }
+                    );
+                } else {
+                    // Remove any accidentally created TestPlanVersion rows created prior to migration
+                    await queryInterface.sequelize.query(
```

Comment:
For reviewers, found an issue during checks against production where several TestPlanVersions were created as duplicates (yet had non-matching `hashedTests`, because of some failed sync between pushes I suppose).

Also broke this out into a separate utility because several migrations may have to make use this in the future as well. Immediately seeing a need for #863 to use this as well.